### PR TITLE
Further cleanup of proxy procedures

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Bringing up proxy registration and procedures up to date, some steps are redundant as they refer to traditional clients or some other details are unclear.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/installation/pages/proxy-registration.adoc [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+After the Salt client is succesfully bootstrapped, it needs to be configured as {productname} Proxy.
 
 This procedure describes software channel setup and registering the installed proxy as the {productname} client, using an activation key.
 
-[IMPORTANT]
-====
-When choosing a username or password for your {productname} Proxy, ensure it does not contain an ``@`` or ``:`` character.
-These characters are reserved.
-====
+//OM: Admonitions decluttering? Do we really need this warning here?
```
Comment (keichwa) [inline modules/installation/pages/proxy-registration.adoc:12]: I've the feeling that it must stay.  But better reach out to the proxy guy :)

File: modules/installation/pages/proxy-setup.adoc [hunk 2]
```diff
@@ -221,25 +219,6 @@ If you do not use the reactivation key, you will need to re-register all the cli
 
 
 
-.Procedure: Replacing a Traditional Proxy and Keeping the Clients Registered
```
Comment (keichwa) [inline modules/installation/pages/proxy-setup.adoc:224]: Do we now have this info elsewhere?  I can imagine that users in the field still have traditionally connected proxies running and want to update or replace them.  But that's just a feeling.

Comment (0rnela) [reply modules/installation/pages/proxy-setup.adoc:224]: I believe as of 4.2 Proxy is purely Salt, so this procedure is redundant. I will double check and request review again. Traditional proxy remains in 4.1 documentation.

Comment (jcayouette) [reply modules/installation/pages/proxy-setup.adoc:224]: Please check, I am not sure about this one either.

**Comment by 0rnela** [comment]
Closing this PR, I'll separate tasks in more manageable way.

**Pull request closed without merging**